__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  db.end();
```

Comment:
You might need a return here if you're getting any messages about things not ending gracefully.